laundry-management-service/api/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'api'
```

Comment:
check how it is done in app module. not sure It's needed.

Reply:
fix